pywbem/cim_operations.py

```diff
@@ -861,6 +890,100 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_server_response_time = None
 
     @property
+    def operation_recorder(self):
+        """
+        This is a writeable property; setting this property will cause the
+        defined recorder to be activated.
+
+        Effective pywbem 0.10.0 operation_recorder became a property rather
+        than an attribute. The getter operation returns TODO
+        The setting operation executes the add_operation_recorder with the
+        value provided. Since there can be multiple simultaneous recorders,
+        setting operation_recorder to None is no longer valid.
+
+
+        **Experimental:** This property is experimental for this release.
+
+        **Deprecated:** This property has been marked deprecated.  The method
+        add_operation_recorder should be used to activate an operation
+        recorder.
+        """
+        warnings.warn(
+            "Directly setting operation recorder has been deprecated",
+            DeprecationWarning)
```

Comment:
1. This is the getter not the setter, so the warning message should be adjusted.
2. The warning message should say what to use instead.

Reply:
Done.

Reply:
Changed to say getter.  Interesting in that there is no replacement.  Once set with add_operation_recorder we do not have a way to remove recorders, just the enable/disable.


DISCUSSION: Should we be able to remove recorders??? or just disable them.

Reply:
I added statement that there is no way to remove once they have been added.

Reply:
Because the performance difference is made by activation, I think we can go for now with not having a remove.